### ISSUE TITLE
latest commit in dev branch fails to run

### ISSUE THREAD

**Issue by andreadega** [sangonzal/repository-traffic-action#23]
Hi there,

Thanks so much for this very useful tool. I would like to use the latest version, which looking at the code seems much improved. However, setting @afe0206f766a8c1b5350977a377bdbcc6ab8072b instead of @v0.1.4 results in the error:

```
 Traceback (most recent call last):
  File "/main.py", line 3, in <module>
    from repostats import RepoStats
ModuleNotFoundError: No module named 'repostats'
```

I wonder if it is just a matter of copying the repostat.py into the docker container, which seems to be missing in the Dockerfile.

**Comment by andreadega** [comment]
An update:

I have forked the latest commit and wrote a few fixes. I'll open a pull request so you can see the changes and whether you'd like to have them merged here.

**Comment by sangonzal** [comment]
Thanks @andreadega! I've merged the changes you proposed and released v0.1.5.
